### PR TITLE
Improve readme for repo

### PR DESCRIPTION
**Pull request by sfc-gh-trichards**
Curious about @sfc-gh-zblackwood's thoughts on this, but just made a quick edit as I'll be sending people towards this link.

### CODE REVIEW

File: samples/streamlit-in-snowflake/sis-deploy/README.md [hunk 1]
```diff
@@ -1,10 +1,15 @@
-# Dependencies
+# Test and Deploy apps with the Snowcli
 
-pip install snowflake-cli-labs==1.2.1
+Often when developing SiS apps, you might want to graduate from the Snowsight UI and begin to develop locally with your preferred development environment (like VSCode), or collaborate with your team on GitHub or another git based environment. This repo provides an example of how to:
+- Set up your Streamlit and Snowflake credentials
+- Install the right dependencies
+- Test and deploy your SiS apps from the command line
+
+If you have issues with this example, feel free to open an issue, talk to your SE, or email me at tyler.richards@snowflake.com. You can find full documentation for the snowcli [here](https://github.com/Snowflake-Labs/snowcli) and specific documentation for the snowcli and SiS [here](https://docs.snowflake.com/LIMITEDACCESS/snowcli/streamlit-apps/overview).
```
Comment (sfc-gh-zblackwood) [inline samples/streamlit-in-snowflake/sis-deploy/README.md:8]: Are we supposed to send out LIMITEDACCESS links?

File: samples/streamlit-in-snowflake/sis-deploy/README.md [hunk 2]
```diff
@@ -30,9 +35,10 @@ schema = "<SCHEMA>"
 warehouse = "<WAREHOUSE>"
 ```
 
-# Set up local conda environment
+# Set up local environment
 
 ```sh
+pip install snowflake-cli-labs==1.2.1
 brew install miniconda`
```
Comment (sfc-gh-zblackwood) [inline samples/streamlit-in-snowflake/sis-deploy/README.md:42]:
```suggestion
brew install miniconda
```

File: samples/streamlit-in-snowflake/sis-deploy/README.md [hunk 3]
```diff
@@ -30,9 +35,10 @@ schema = "<SCHEMA>"
...
 conda env update`
```
Comment (sfc-gh-zblackwood) [inline samples/streamlit-in-snowflake/sis-deploy/README.md:43]:
```suggestion
conda env update
```

File: samples/streamlit-in-snowflake/sis-deploy/README.md (hunk 2 above)
Comment (sfc-gh-zblackwood) [reply samples/streamlit-in-snowflake/sis-deploy/README.md:42]: Could fix these while you're in there

File: samples/streamlit-in-snowflake/sis-deploy/README.md [hunk 4]
```diff
@@ -30,9 +35,10 @@ schema = "<SCHEMA>"
...
 conda activate sis-deploy`
```
Comment (sfc-gh-zblackwood) [inline samples/streamlit-in-snowflake/sis-deploy/README.md:44]:
```suggestion
conda activate sis-deploy
```

File: samples/streamlit-in-snowflake/sis-deploy/README.md (hunk 2 above)
Comment (sfc-gh-zblackwood) [inline samples/streamlit-in-snowflake/sis-deploy/README.md:42]: Might want to note that these are the instructions for Mac/Linux, and will be different on Windows

**Review by sfc-gh-zblackwood — APPROVED** [review]
A couple small notes, but this seems like a good improvement

File: samples/streamlit-in-snowflake/sis-deploy/README.md (hunk 1 above)
Comment (sfc-gh-trichards) [reply samples/streamlit-in-snowflake/sis-deploy/README.md:8]: yeah it should be fine

**Comment by sfc-gh-trichards** [comment]
@sfc-gh-ghernandez do you know the best person to review this who has the right authorization?

**Review by sfc-gh-ghernandez — APPROVED** [review]
Approved given @sfc-gh-trichards and @sfc-gh-zblackwood are discussing and peer reviewing.

**Comment by sfc-gh-ghernandez** [comment]
> @sfc-gh-ghernandez do you know the best person to review this who has the right authorization?

Yep, I approved and left a note. Thanks @sfc-gh-trichards

**Comment by sfc-gh-trichards** [comment]
@sfc-gh-ghernandez would you mind merging too? It says I cannot merge

**Review by sfc-gh-ghernandez — APPROVED** [review]

**Comment by sfc-gh-ghernandez** [comment]
> @sfc-gh-ghernandez would you mind merging too? It says I cannot merge

Yep, merged!

**Pull request merged**